.gitignore

```diff
@@ -2,3 +2,4 @@
 dist/
 node_modules/
 target/
+.oasis/
```

Comment:
what generates this?

Reply:
I'm not sure but it was generated automatically from building/running the tests.